pkg/Microsoft.Private.PackageBaseline/stable.packages.props

```diff
@@ -1397,5 +1397,46 @@
     <StablePackage Include="runtime.ubuntu.16.04-x64.runtime.native.System.Security.Cryptography">
       <Version>1.0.2</Version>
     </StablePackage>
+
+    <!-- Servicing 1.0.6 wave -->
+    <StablePackage Include="Microsoft.NETCore.Targets">
+      <Version>1.0.4</Version>
+    </StablePackage>
+    <StablePackage Include="System.IO.FileSystem.DriveInfo">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Http">
+      <Version>4.1.3</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Http.WinHttpHandler">
+      <Version>4.0.4</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Security">
+      <Version>4.0.2</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.debian.8-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.fedora.23-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.native.System.Security.Cryptography">
+      <Version>4.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.opensuse.13.2-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.rhel.7-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.ubuntu.14.04-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.ubuntu.16.04-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.3</Version>
+    </StablePackage>
```

Comment:
Might be interesting to just point the index update task at something like https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/1.0.0/Latest_Packages.txt in order to semi-automate these changes.